src/client/dfs/dfs.c

```diff
@@ -2830,7 +2830,7 @@ dfs_global2local(daos_handle_t poh, daos_handle_t coh, int flags, d_iov_t glob,
 	strcpy(dfs->root.name, "/");
 	dfs->root.mode = S_IFDIR | 0755;
 
-	obj_mode = get_daos_obj_mode(flags);
+	obj_mode = get_daos_obj_mode(flags ?: dfs_params->amode);
```

Comment:
i believe this is a C extension? for portability and clarity i would prefer you specify the second operand explicitly.

Reply:
oh, sure.